.github/workflows/publish.yml

```diff
@@ -101,7 +146,7 @@ jobs:
 
   tagMilestone:
     name: Tag milestone
-    needs: [ prepare, deployMilestone ]
+    needs: [ reactor-netty-core, reactor-netty-http, reactor-netty-http-brave, reactor-netty-incubator-quic, deployMilestone ]
```

Comment:
I think that just `needs: [ deployMilestone, reactor-netty-core ]` would be enough since the deploy job itself needs all these others test-only jobs. it still needs to reference `reactor-netty-core` because it uses its output below in the tag message.

Reply:
fixed